lib/Ride_History.dart

```diff
@@ -0,0 +1,188 @@
+import 'package:flutter/material.dart';
+
+class RideHistory extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+
+    final directionStyle = TextStyle(
+        color: Colors.grey[500],
+        fontWeight: FontWeight.w500,
+        letterSpacing: 0.2,
+        fontSize: 12,
+        height: 2
+    );
+
+    final infoStyle = TextStyle(
+        color: Colors.black,
+        fontWeight: FontWeight.w400,
+        letterSpacing: 0.2,
+        fontSize: 18,
+        height: 2
+    );
+
+    final monthStyle = TextStyle(
+        color: Colors.black,
+        fontWeight: FontWeight.w700,
+        letterSpacing: 0.2,
+        fontSize: 18,
+        height: 2
+    );
+
+    final dayStyle = TextStyle(
+        color: Colors.grey[700],
+        fontWeight: FontWeight.w500,
+        letterSpacing: 0.2,
+        fontSize: 18,
+        height: 2
+    );
+
+    final timeStyle = TextStyle(
+        color: Colors.grey[500],
+        fontWeight: FontWeight.w400,
+        letterSpacing: 0.2,
+        fontSize: 17,
+        height: 2
+    );
+
+    return Column(
+      children: <Widget>[
+        Container(
+          margin: EdgeInsets.only(left: 17.0, right: 17.0, bottom: 15.0),
+          width: 400.0,
+          height: 125.0,
```

Comment:
Do not set absolute size. Content will overflow containers. Instead use relative sizing by using MediaQuery.of(context).size.width and .height